spec/providers/coreIntegration/tcpsocket.integration.src.js

```diff
@@ -131,7 +167,7 @@ module.exports = function (provider, setup) {
 
         // Wait a second before unpausing.  Data received in this second
         // will fail the expectation that paused is false in gotMessageAsync.
-        setTimeout(function() {
+        setTimeout(function () {
```

Comment:
this seems inherently flaky. can we
1. open server, client
2. pause server
3. send data from client
4. check not received on server
5. unpause server
6. check now received on server